<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registro</class>
 <widget class="QMainWindow" name="Registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>497</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>706</width>
    <height>497</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>706</width>
    <height>497</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Guiñote</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-color: rgb(0, 85, 0);
}

#formLayout{
background-color: rgb(255,255,255)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>242</y>
      <width>581</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <property name="verticalSpacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_nombre_real">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre real:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="text_nombre_real"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_nombre_usuario_registro">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre de usuario:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="text_nombre_usuario_registro"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_correo">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Correo electrónico:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="text_correo"/>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="boton_terminos">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Acepto los términos y condiciones</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="boton_confirmar_registro">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Registrarse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="text_contrasenya_registro"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_contrasenya_registro">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Contraseña:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/logo/150.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
